File: Assignment1/codes/allterms.py
import math

def x_n(n):
    if n <= 0:
        return 0
    else:
        return (-1) / math.factorial(n)

# Get N from the user
print("//This program will generate the terms of the sequence upto x(n)// \n")
N = int(input("Enter the value of n: "))

# Generate the sequence up to N

print("...\nx( -1 ) = 0\n")
for x in range(N+1):
    {    
        print("x(", x, ") = ", x_n(x), "\n")
    }
print("...")

File: Assignment2/codes/allterms.c
#include <stdio.h>

int main() {
    int n, N;
    printf("\\This program will print the values of the terms of the sequence upto x(n)\\\n");
    printf("Enter value of n: ");
    scanf("%d", &N);

    // Printing the series header
    printf("... \nx(0) = 0\n");

    // Calculate and print the terms of the series for n = 0 to 10 (you can change the range as needed)
    for (n = 1; n <= N; ++n) {
        // Print the result in fraction format
        printf("x(%d) = %lld/4\n", n, (n+2)*(n+2));
    }

    return 0;
}

